package.json

```diff
@@ -79,5 +79,8 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org/"
+  },
```

Comment:
Can you remove this? No need for a dependency here

Reply:
lol not even sure how that got changed since i didn't even open that file....but cool. Should be fixed now